fixed-price-subscriptions/client/react/src/Account.js

```diff
@@ -1,196 +1,65 @@
 import React, { useState, useEffect } from 'react';
-import { withRouter } from 'react-router-dom';
+import { Link, withRouter } from 'react-router-dom';
 import './App.css';
-import TopNavigationBar from './TopNavigationBar';
-import StripeSampleFooter from './StripeSampleFooter';
-import Product from './Product';
-import PriceChangeForm from './PriceChangeForm';
 
-const products = [
-  {
-    key: 0,
-    price: '$5.00',
-    name: 'Basic',
-    interval: 'month',
-    billed: 'monthly',
-  },
-  {
-    key: 1,
-    price: '$15.00',
-    name: 'Premium',
-    interval: 'month',
-    billed: 'monthly',
-  },
-];
+const AccountSubscription = ({subscription}) => {
+  return (
+    <section>
+      <hr />
+      <h4>
+        <a href={`https://dashboard.stripe.com/test/subscriptions/${subscription.id}`}>
+          {subscription.id}
+        </a>
+      </h4>
 
-function Account({ location }) {
-  console.log(location);
-  const [accountInformation] = useState(location.state.accountInformation);
-  let [customerPaymentMethod, setCustomerPaymentmethod] = useState(null);
-  let [showChangePriceForm, setShowChangePriceForm] = useState(false);
-  let [subscriptionCancelled, setSubscriptionCancelled] = useState(false);
-  let [newProductSelected, setNewProdctSelected] = useState('');
-  let [selectedProducted, setSelectedProduct] = useState(
-    accountInformation.priceId
-  );
+      <p>
+        Status: {subscription.status}
+      </p>
 
-  useEffect(() => {
-    async function fetchData() {
-      // You can await here
-      const response = await fetch('/retrieve-customer-payment-method', {
-        method: 'post',
-        headers: {
-          'Content-type': 'application/json',
-        },
-        body: JSON.stringify({
-          paymentMethodId: accountInformation.paymentMethodId,
-        }),
-      });
-      const responseBody = await response.json();
-      const paymentMethod =
-        responseBody.card.brand + ' •••• ' + responseBody.card.last4;
+      <p>
+        Card last4: {subscription.default_payment_method.card.last4}
+      </p>
 
-      setCustomerPaymentmethod(paymentMethod);
-    }
-    fetchData();
-  }, [accountInformation.paymentMethodId]);
+      <p>
+        Current period end: {(new Date(subscription.current_period_end * 1000).toString())}
+      </p>
 
-  function handleChangePriceForm() {
-    setShowChangePriceForm(true);
-  }
+      {/* <Link to={{pathname: '/change-plan', state: {subscription: subscription.id }}}>Change plan</Link><br /> */}
```

Comment:
OOC, why the commented out link to `/change-plan`?

Reply:
Because it's not implemented :) I want to see if there's appetite for it first.